pyproject.toml

```diff
@@ -9,13 +9,15 @@ requires = [
 
 [tool.towncrier]
     # Do no set this since it is hard to import numpy inside the source directory
-    # Use "--version Numpy" instead
-    #project = "numpy"
-    filename = "doc/release/latest-note.rst"
-    directory = "doc/release/upcoming_changes"
+    # the name is hardcoded. Use "--version 1.18.0" to set the version
+    single_file = false
+    filename = "doc/source/release/{version}-notes.rst"
```

Comment:
this gives us a live view of the release note, nice!

Reply:
ahh, except it doesn't. CircleCI says
```
/home/circleci/repo/doc/source/release/{version}-notes.rst: WARNING: document isn't included in any toctree
```

Reply:
Yeah, the `single_file` option is opposit to what I thought :), I think hawkowl changed it from my PR, so it created the wrong file. It will create a live view of course.